engine/src/main/java/org/terasology/rendering/nui/layouts/CardLayout.java

```diff
@@ -26,20 +26,43 @@
 import java.util.Iterator;
 import java.util.Map;
 
+/**
+ * A layout that allows for a single widget to be displayed among a list of stored widgets
+ */
 public class CardLayout extends CoreLayout<LayoutHint> {
+    /**
+     * The id of the currently displayed widget
+     */
```

Comment:
Thank you for the Ctrl-Q tip: I didn't know about it and it's a neat feature.

I remain unconvinced about private variables having a javadoc instead of a simple comments. The html javadocs are usually generated without private fields and methods, so, I guess that's not a problem. But I think there should be a clear distinction between API-level documentation, based on javadocs, and source-level documentation based on standard comments. If anything, and given you were talking about consistency, I don't think this practice would be consistent with the rest of Terasology's code. 

But there are many areas in the codebase I have not visited. So, I could be wrong.